en/guide/define_xml_element.md

```diff
@@ -222,16 +225,42 @@ Care should still be taken to ensure that any changes that alter the way that th
 Messages are very rarely deleted, as this may break compatibility with legacy MAVLink 1 hardware that is unlikely to be updated to more recent versions.
 
 
-#### MAVLink 2 Message Extensions {#message_extensions}
+### Message Extensions (MAVLink 2) {#message_extensions}
```

Comment:
MOved the doc on extensions here into section on where you declare them - which seems more logical than having them in a MAVLink 2 specific section.